plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/FastXSLTMediatorPropertiesEditionPart.java

```diff
@@ -91,8 +79,30 @@
 	public void setFastXsltSchemaKeyType(Enumerator newValue);
 
 
+	/**
+	 * @return the description
+	 * 
+	 */
+	public String getDescription();
+
+	/**
+	 * Defines a new description
+	 * @param newValue the new description to set
+	 * 
+	 */
+	public void setDescription(String newValue);
 
+	// Start of user code for fastXsltStaticSchemaKey specific getters and setters declaration
+	public RegistryKeyProperty getFastXsltStaticSchemaKey();
 
+    public void setFastXsltStaticSchemaKey(RegistryKeyProperty fastXsltStaticSchemaKey);
```

Comment:
Fix formatting issue (#98-#105)

Reply:
Fixed in https://github.com/wso2/devstudio-tooling-esb/pull/804